src/components/tooltip/Marker.js

```diff
@@ -312,12 +312,15 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
         <TooltipDetailSeparator key="backtrace-separator" />,
         <TooltipDetail label="Stack" key="backtrace">
           <div className="tooltipDetailsBackTrace">
-            {data.type === 'Styles' || marker.name === 'Reflow' ? (
-              <h2 className="tooltipBackTraceTitle">
-                First invalidated {formatNumber(causeAge)}ms before the flush,
-                at:
-              </h2>
-            ) : null}
+            <h2 className="tooltipBackTraceTitle">
+              {data.type === 'Styles' || marker.name === 'Reflow'
+                ? `First invalidated ${formatNumber(
+                    causeAge
+                  )}ms before the flush,
+                at:`
```

Comment:
Since we changed this to use template strings, this is now also adding new line character after `flush,` it's probably not a huge problem but it would be better to move this `at:` to the line above.
So the above line can be like this `)}ms before the flush, at:`